test/automation/search_automation_test.go

```diff
@@ -0,0 +1,128 @@
+package automation
+
+import (
+	"fmt"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
+	"github.com/stretchr/testify/suite"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+type SearchAutomationTestSuite struct {
+	tagsuite.Suite
+}
+
+func (suite *SearchAutomationTestSuite) TestSearch_NoArg() {
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("search")
+	cp.ExpectLongString("The following argument is required:")
+	cp.ExpectExitCode(1)
+
+	cp = ts.Spawn("search", "--language", "python")
+	cp.ExpectLongString("The following argument is required:")
+	cp.ExpectExitCode(1)
+
+}
+
+func (suite *SearchAutomationTestSuite) TestSearch_NoLanguageArg() {
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("search", "--language")
+	cp.ExpectLongString("flag needs an argument: --language")
+	cp.ExpectExitCode(1)
+
+}
+
+func (suite *SearchAutomationTestSuite) TestSearch_OutProject() {
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.Spawn("search", "flask")
+	cp.ExpectLongString("Language must be provided by flag or by running this command within a project.")
```

Comment:
I would recommend shrinking this assertion to something like:

```
cp.ExpectLongString("Language must be provided by flag")
```

Two reasons for this:

1. The longer the assertion the more likely we'll run into text wrapping issues in the assertion
2. The longer the assertion the more likely it is to break if we make copy changes